qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateDeploymentTest.java

```diff
@@ -139,7 +139,7 @@ public void shouldRejectDeployIfResourceIsTooLarge() {
     final var modelThatFitsJustWithinMaxMessageSize =
         Bpmn.createExecutableProcess("PROCESS")
             .startEvent()
-            .documentation("x".repeat((1047100)))
+            .documentation("x".repeat((1046900)))
```

Comment:
🔧 It would be nice if you add a body to this commit message explaining why this change was necessary. Why did the test fail and why did this resolve it?

From context I assume it is because the new attributes have changes the size of the deployment, so now it needs to be dropped slightly so it still fits.

Reply:
Yes, you are right. Sorry for that!